CONTRIBUTING.md

```diff
@@ -41,9 +76,9 @@ important that you create an issue as this can be more easily overlooked.
 
 When coding, please also add a string constant for your strings to ```./src/localisation/StringIds.h```.
 
-### Coding style
+## 5.2 Coding style
 Use [this](https://github.com/OpenRCT2/OpenRCT2/wiki/Coding-Style) code style as a reference for new or changed code.
 
-### Language
+## 5.3 Language
 For now, it is recommended that you only write C++ files as the majority of the game is currently in
 C++. Exceptions are to modules that have direct relationship to original code.
```

Comment:
I think this whole section could be dropped, or rewritten to say it's C++ only except when dealing with the plugins files?

What do you think @OpenRCT2/developers ?